rust/benches/benches/models/parse_models.rs

```diff
@@ -0,0 +1,44 @@
+#![allow(non_snake_case)]
+
+mod utils;
+pub use utils::*;
```

Comment:
`pub use` is a public re-export of the named modules, which is usually not what's needed in internal modules (but it has of course useful applications). i'd suggest to just `use utils;`.

as a side note, in the current way of declaring it this `pub use` is completely unused, because the methods are called as `utils::...` which is possible due to line 3 above: `mod utils;`. clippy just doesn't complain about unused functions because being unused is expected for public re-exports.

this is probably also the reason why you have seen the unused-functions-warning.

Reply:
i took it from [here](https://github.com/rust-lang/rust/issues/46379#issuecomment-487556641). i stopped getting unused warnings when i added this line, but it's just a hack, i guess

i'll address the project structure in another reply

Reply:
changed.